pkg/redirectpolicy/manager.go

```diff
@@ -110,6 +114,24 @@ func (rpm *Manager) RegisterGetStores(sg StoreGetter) {
 // AddRedirectPolicy parses the given local redirect policy config, and updates
 // internal state with the config fields.
 func (rpm *Manager) AddRedirectPolicy(config LRPConfig) (bool, error) {
+	rpm.warnOnce.Do(func() {
+		if option.Config.EnableLocalRedirectPolicy {
```

Comment:
nit, is this check necessary? Sure `AddRedirectPolicy()` should not be called if the feature is disabled?

Reply:
Yes, agreed.